### DISCUSSION TITLE
How to add new option Sort By Position in frontend category product list page?

### DISCUSSION THREAD

**Discussion by khanriyaz617** [bagisto/bagisto#7644] [Q&A]
**Hi Developers,**

**I have a requirement where, I am providing option in backend to re-position products in sortable form using drag and drop for Category Men or whaever it be. I have done this part and already have stored current updated product position in database.**

But Now, I want to add a new option (see below screenshot) **Sort By Position** in Sort By column and then product should list by its position.
![WhatsApp Image 2023-06-11 at 19 38 11](https://github.com/bagisto/bagisto/assets/10295136/80b04cbe-ae00-4a67-a418-c1c76e54dd41)

**Actually I am appyling it in velocity theme but I have less idea behind Bagisto's frontend part execution. Anybody can explain me how it will be achieved?**

**Thank in Advance.**

**Comment by amit-webkul** [comment]
Hi @Riyazkhan1989 ,

If you want to add a new customer filter option to the Velocity theme, you'll need to make some code changes within the admin package. Follow these steps to add your custom option:

Navigate to the following folder path: Package -> Webkul -> Admin -> src -> config -> system.php.

Inside this directory, you'll find the 'sort_by' key. You can add your custom filter option here.

Once you've completed these steps, check your admin panel in the Product Configuration section. Your custom option should now be visible and ready for use according to your preferences.

**Comment by Amitk-webkul** [comment]
Hi @Riyazkhan1989, I believe you've found your solution. If you have any more questions in the future, don't hesitate to reach out. Just go ahead and close this discussion for now.
